src/main/java/bio/terra/app/model/GoogleCloudResource.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.app.model;
+
+/**
+ * Google cloud resources used by TDR.
+ */
+public enum GoogleCloudResource {
+    BIGQUERY("bigquery"), FIRESTORE("firestore"), BUCKET("bucket");
+
+    private String value;
```

Comment:
All enum fields should be `final`
```suggestion
    private final String value;
```

Reply:
how can this be final if it needs to be referenced in the enum constructor?

Reply:
WAIT STATIC NOT FINAL DOH